### ISSUE TITLE
New Error

### ISSUE THREAD

**Issue by J-Durham** [honestbleeps/Reddit-Enhancement-Suite#5357]
![image](https://user-images.githubusercontent.com/53316234/143991269-7234b38f-2dc5-430b-af38-89851855f35b.png)
just got this error, don't know how or if I can duplicate it. Just wanted to let you know that there is a logical branch that can throw this error

**Comment by creesch** [comment]
That's probably a reddit error and nothing to do with RES

**Comment by hlovdal** [comment]
I agree, this really seems like a reddit error. @J-Durham, next time if you open the developer console (F12 on Firefox/Chrome, Ctrl+Shift+I on Opera) there will most likely be some logging there that will provide some context to what happened.
